validators/cgroup_validator_other.go

```diff
@@ -23,6 +23,8 @@ package system
 
 var _ Validator = &CgroupsValidator{}
 
+const mountsFilePath = ""
```

Comment:
this is not needed? the definition of mountsFilePath is only required in the linux file, as far as i can tell.

Reply:
never mind, it's referenced in validators/kernel_validator.go